sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/DataSourceSuite.scala

```diff
@@ -142,7 +142,8 @@ class DataSourceSuite extends SharedSparkSession with PrivateMethodTester {
     val message = intercept[java.io.IOException] {
       dataSource invokePrivate checkAndGlobPathIfNecessary(false, false)
     }.getMessage
-    assert(message.equals("No FileSystem for scheme: nonexistsFs"))
+    val expectMessage = "No FileSystem for scheme nonexistFS"
```

Comment:
`nonexistFS` -> `nonexistsFs`?

Reply:
well, then I would prefer nonExistingFS..I was trying to keep the naming simply.
Let me change them all since you are asking.

Reply:
Ur, I asked this because this test case fail still.
```
[info] - Data source options should be propagated in method checkAndGlobPathIfNecessary *** FAILED *** (599 milliseconds)
[info]   "... for scheme nonexist[sFs]" did not equal "... for scheme nonexist[FS]" (DataSourceSuite.scala:146)
```

Reply:
Actually, I don't care about the naming here if it passes with `-Phadoop-3.2`.

Reply:
I see :)